File: Python/PGS/LEVEL1/두 정수 사이의 합/gunwoo.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int change = 0;
        if(a > b){
            change = a;
            a = b;
            b = change;
        }
        for(int i = a; i <= b; i++){
            answer += i;
        }
        return answer;
    }
}

File: Algorithm/PGS/LEVEL1/삼총사/gunwoo.py
from itertools import combinations as comb

def solution(number):
    answer = 0
    com = list(comb(number, 3))
    for i in range(len(com)):
        if sum(com[i]) == 0:
            answer +=1
            
    return answer

#combinations 알면 쉬운 문제